src/_store/configureStore.js

```diff
@@ -6,7 +6,9 @@ import createEngine from 'redux-storage/engines/localStorage';
 import * as ActionTypes from '../_constants/ActionTypes';
 
 const actionsToCache = [
-    ActionTypes.SIGNIN_FIELD_UPDATE,
+    ActionTypes.ADD_TOKEN,
+    ActionTypes.REMOVE_TOKEN,
+    ActionTypes.SERVER_AUTH_FAILED,
     ActionTypes.SERVER_DATA_ASSET_INDEX,
```

Comment:
Why? we are not caching signin as a whole, which should we trigger localstorage save when signin_field_update happens?